### PR TITLE
Disable liveness for dns due to #10659

### PR DESCRIPTION
**Pull request by dchen1107**
cc/ @alex-mohr @bprashanth @thockin

### CODE REVIEW

File: cluster/addons/dns/skydns-rc.yaml.in [hunk 1]
```diff
@@ -61,12 +61,4 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
-        livenessProbe:
```
Comment (thockin) [inline cluster/addons/dns/skydns-rc.yaml.in]: You also have to update the 'v4' all over to be 'v5'

Comment (dchen1107) [inline cluster/addons/dns/skydns-rc.yaml.in]: We only change manfest file here, why requires version change here? If it does, we already messed up with https://github.com/GoogleCloudPlatform/kubernetes/commit/988aa6fdf67110cdfc012bf79a15d39c337f58bc then.

**Comment by dchen1107** [comment]
Rebased and update it to v6 now

**Comment by alex-mohr** [comment]
@zmerlynn fyi, cherrypick for 0.21.1.

File: cluster/addons/dns/skydns-rc.yaml.in (hunk 1 above)
Comment (thockin) [inline cluster/addons/dns/skydns-rc.yaml.in]: yeah, but the manifest is materially different so we want a version on it for now.

**Comment by thockin** [comment]
Approving because of urgency.

@yujuhong please give priority to this one

**Comment by yujuhong** [comment]
Will merge on green

**Pull request merged**